<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>392</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Organization:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Room: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Nature of Activity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Time Start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Time End:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="lineEdit_organization"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_room">
           <item>
            <property name="text">
             <string>AVR1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AVR2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AVR3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Plenary Hall</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Seminar Room</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gym</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_natureOfActivity">
           <item>
            <property name="text">
             <string>Seminar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Orientation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Meeting</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_timeStart">
           <item>
            <property name="text">
             <string>07:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19:30</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_timeEnd">
           <item>
            <property name="text">
             <string>09:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18:00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19:30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21:00</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reserve">
         <property name="text">
          <string>Reserve</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget_schedule">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>911</width>
      <height>371</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7:30 - 9:00 AM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9:00 - 10:30 AM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10:30 - 12:00 NN</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12:00 - 1:30 PM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1:30 - 3:00 PM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3:00 - 4:30 PM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4:30 - 6:00 PM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7:30 - 9:00 PM</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Mon</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tue</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wed</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thu</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sun</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
